understand-conten-marketing/readme.md

```diff
@@ -0,0 +1,76 @@
+> Goal of this guide: to help you understand what content marketing is.
+
+# What the flying flop is content marketing?
+
+In *The Rise of Storytelling as the New Marketing*, Joe Pulizzi describes content marketing as “the idea that storytelling is key to attract[ing] and retain[ing] customers.”
+
+The first known example of content marketing was the creation of a magazine, first published in 1895 by John Deere (a company that sells tools for farmers), called *the Furrow*. It’s goal was not to sell farm equipment, but to provide knowledge on farming, making John Deere an expert on farming.
+
+By doing this, John Deere gave valuable information to their readers. The value that the readers received from the magazine was directly linked to the sender, John Deere. The value the reader now linked to John Deere greatly impacted how they valued all other things John Deere did, including selling farm equipment.
+
+And sure enough, when those farmers needed new gear they would go to the expert they knew and trusted: John Deere.
+
+The next few quotes are from the *Journal of Marketing Management* in the article “Understanding Digital Content Marketing”, in which Jenifer Rowley describes some key notions about content marketing:
+
+“...a recurrent theme in DCM (digital content marketing) is customer value, and this leads into the second major contribution of the article, an exploration of the notion of customer value in digital content marketplaces.”
+
+“The difficulty in developing a notion of 'value' in the context of digital content”
+
+“The customer's experience of digital content is influenced by all stakeholders in the value chain as well as requiring the customer to participate in learning and co-creation of the experience.”
+
+“The potential of the 'value-in-use' notion of customer value as a basis for ensuring that consumers are fully engaged as stakeholders in the content economy.”
+
+## What you need to know in order to make awesome content marketing
+
+In the first part of the guide, we used a bunch of quotes to help you understand what content marketing is and what its struggles are. In the rest of the guide, we will try to explain a few concepts which might help you create awesome content marketing. Everything beyond this point is our own interpretation and by no means an unwavering truth.
```

Comment:
The John Deere bit is awesome and supports the first sentence here. Unfortunately, the quotes from the *Journal of Marketing Management* confuse rather than help understanding. Also, should this sentence go before the header? Seems funny here...